python/tests/core/test_transformation_function_engine.py

```diff
@@ -27,48 +29,49 @@
 from hsfs.hopsworks_udf import UDFType, udf
 
 
-fg1 = feature_group.FeatureGroup(
-    name="test1",
-    version=1,
-    featurestore_id=99,
-    primary_key=[],
-    partition_key=[],
-    features=[
-        feature.Feature("id"),
-        feature.Feature("label"),
-        feature.Feature("tf_name"),
-    ],
-    id=11,
-    stream=False,
-)
-
-fg2 = feature_group.FeatureGroup(
-    name="test2",
-    version=1,
-    featurestore_id=99,
-    primary_key=[],
-    partition_key=[],
-    features=[feature.Feature("id"), feature.Feature("tf1_name")],
-    id=12,
-    stream=False,
-)
-
-fg3 = feature_group.FeatureGroup(
-    name="test3",
-    version=1,
-    featurestore_id=99,
-    primary_key=[],
-    partition_key=[],
-    features=[
-        feature.Feature("id"),
-        feature.Feature("tf_name"),
-        feature.Feature("tf1_name"),
-        feature.Feature("tf3_name"),
-    ],
-    id=12,
-    stream=False,
-)
-engine.init("python")
+with mock.patch("hopsworks_common.client.get_instance"):
```

Comment:
Is this relevant to this PR?

Reply:
Yes, since `FeatureGroup.__init__` calls `VariableApi.__init__`.